src/main/java/org/isf/utils/jobjects/InventoryState.java

```diff
@@ -0,0 +1,55 @@
+/*
+ * Open Hospital (www.open-hospital.org)
+ * Copyright © 2006-2024 Informatici Senza Frontiere (info@informaticisenzafrontiere.org)
+ *
+ * Open Hospital is a free and open source software for healthcare data management.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * https://www.gnu.org/licenses/gpl-3.0-standalone.html
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see <https://www.gnu.org/licenses/>.
+ */
+package org.isf.utils.jobjects;
+
+
+public class InventoryState {
+	public InventoryState(){}
+	public enum State
+	 {		
+		PROGRESS ("1", "angal.common.statecanceled"),
+		CANCELED ("2", "angal.common.stateinprogress"),
+		VALIDATE ("3", "angal.common.statevalidate");
```

Comment:
the mapping looks wrong here...
Anyway I think that PROGRESS could be DRAFT
Then status for VALIDATE should be VALIDATED (past participle)

Reply:
so I should update  stateinprogress to statedraft  and angal.common.statevalidate to angal.common.statevalidated. right?

Reply:
for LaboratoryStatus we did just like this. 
If the lowercase part is the one that will go in the database, we SHOULD NOT use translations here.
```
package org.isf.lab.model;

public enum LaboratoryStatus {
	draft, open, done, invalid, deleted;
}
```
The statuses you planned for inventory are instead: `draft`, `validated` and `canceled`, correct? ("draft" is to keep consistency with other models in CORE).
Then, translations are related to GUI, and I guess you should use `angal.medicalstock.inventory` key or similar.